test/test.js

```diff
@@ -726,7 +726,7 @@ var testSearch = function(testPath) {
     Async.parallel([
       function(done) {
         request(app)
-          .get(`${testPath}?date__gte=${dateValue}`)
```

Comment:
Why were the tests modified. Please verify that this does not break reverse compatibility.